Readme update

marekksiazek opened the pull request:
## Summary

Update readme

## Details

Change README content. Add some items like badges and change text.

marekksiazek committed:
Update readme.md

StraykerPL commented:
Some changes are shared with other branch, please watch out for this situation in future as this is dangerous:
![image](https://user-images.githubusercontent.com/45916090/220746748-c91d70b1-eb7f-4a25-b54c-dee4e850c508.png)

StraykerPL commented:
Hehe XD
![image](https://user-images.githubusercontent.com/45916090/220747317-325c701f-3bda-4d7e-b776-a0f4ff834c72.png)

StraykerPL commented on README.md:
```diff
@@ -1,19 +1,36 @@
+    <img src="https://img.shields.io/badge/Coverage-0%25-lightgrey" alt="version_badge" width="120px" height="120px"/>
+    <img src="https://img.shields.io/badge/Platform-windows%20%7C%20macos%20%7C%20linux-lightgrey" alt="version_badge" width="120px" height="120px"/>
+</p>
+Cross-platform solution for **task-oriented data management**, supporting **Windows, Linux, macOS, Android and iOS**.  
```
This formatting is not working:
![image](https://user-images.githubusercontent.com/45916090/220747463-5b57519d-7f8d-40ff-8122-0489543ddfc1.png)

marekksiazek committed:
Update README.md

marekksiazek reviewed (commented):
Corrected

marekksiazek committed:
Merge branch 'develop' into readme_update

StraykerPL commented on README.md:
```diff
@@ -1,19 +1,37 @@
-# Contact
+## Documentation
+
+- [Getting Started][quickstart]
```
Link redirection syntax in Markdown is: [My Website](https://straykersoftware.pl)

StraykerPL replied:
Check once more the order of elements in ticket and rearrange them in ReadMe

StraykerPL replied:
Every dot in ticket is one section in ReadMe, sections' titles should be in table of content.

StraykerPL replied:
Tech Stack is missing.

StraykerPL replied:
Contribution Guidelines are missing.

marekksiazek committed:
Update README.md

Add Table of content, Tech stack, getting started, contribution guidelines and update contact section

marekksiazek reviewed (commented):
Corrected

marekksiazek committed:
Delete test_request.md

marekksiazek committed:
Delete refactor_request.md

marekksiazek committed:
Merge branch 'develop' into readme_update
